frontend/simple/views/PayGroup.vue

```diff
@@ -1,62 +1,314 @@
 <template>
   <main>
-    <section class="section">
-      <div class="columns is-multiline">
-        <div class="column is-half">
-          <p class="title is-4"><i18n>{{msg}}</i18n></p>
-          <div class="box">
-            <p>Mincome: <b>${{minCome}}</b></p>
-            <p>Amount the group received this month: <b>${{amountReceivedThisMonth}}</b></p>
-          </div>
-          <div class="box">
-            <p>
-              Contribution Amount:
-              <input id="contributionAmount" class="input" type="text" placeholder="0">
-            </p>
-            <p class="select">
-              <select data-vv-rules="required" name="contributionCurrency">
-                <option value="usd" selected>USD</option>
-                <option value="btc">BTC</option>
-              </select>
-            </p>
-          </div>
-        </div>
+    <div class="section is-hero">
+      <h1 class="title is-1 is-marginless">
+        <i18n>Pay Group</i18n>
+      </h1>
+      <p><i18n>What’s this page about, so the user understands the context.</i18n></p>
+    </div>
+
+    <section class="section columns is-desktop is-multiline c-invoice">
+      <header class="box column is-narrow is-flex-touch gi-is-justify-between c-summary">
+        <div class="c-summary-item">
+          <h2 class="is-size-7 has-text-grey is-uppercase"><i18n>Payments Sent</i18n></h2>
+          <p class="title is-5">
+            {{paymentSummary.sent}} <i18n>of</i18n> {{users.length}}
```

Comment:
This should all be enclosed within the `<i18n>` tag, and using the template mechanism that @vijayee introduced in a [previous PR](https://github.com/okTurtles/group-income-simple/pull/469) where args are passed in via `<i18n :args="object"> .. </i18n>`.

See Victor's comment: https://github.com/okTurtles/group-income-simple/issues/417#issuecomment-411111877

Reply:
Updated to use `i18n`. But a really strange bug happens... I didn't debug it, meanwhile you can also take a look at it:

![aug-28-2018 20-04-28](https://user-images.githubusercontent.com/14869087/44744705-a4046780-aafd-11e8-9a19-1f833fd2722b.gif)